examples/http-streaming/http_streaming.server.out

```diff
@@ -1,4 +1,4 @@
 #At the command line, navigate to the directory that contains the 
 #`.bal` file and run the `ballerina run` command. 
 $ ballerina run http_streaming.bal
-
+ballerina: started HTTP/WS endpoint 0.0.0.0:9090
```

Comment:
shouldn't the complete output be the following:
```
Initiating service(s) in 'http_streaming.bal'
ballerina: started HTTP/WS endpoint 0.0.0.0:9090
```
Same applies to the other two as well